### PR TITLE
Update prereq-overview.md

### PR DESCRIPTION
**Pull request by DominicWatts**
## Purpose of this pull request

Clarify MySQL versions for 2.3

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/install-gde/prereq/prereq-overview.html#mysql

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/install-gde/prereq/prereq-overview.md
Comment (BarnyShergold) [inline src/guides/v2.3/install-gde/prereq/prereq-overview.md:86]: I would query the statement that Magento has not been tested on MariaDB 10.1 - Magento Cloud runs on MariaDB 10.1 / 10.2 : https://devdocs.magento.com/cloud/project/services-mysql.html

I feel that this needs to be justified.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/install-gde/prereq/prereq-overview.md
Comment (jeff-matthews) [reply src/guides/v2.3/install-gde/prereq/prereq-overview.md:86]: I think it's best defer to the main system reqs page instead of repeating versions in different places; it makes maintenance really difficult. The main system reqs page is the source of truth about what versions of third-party software Magento Commerce and Open Source have been tested with. These may not be the same as Magento Cloud.

Also, we have a sentence at the top of the 2.4 main system requirements page that directs users with cloud projects to a different service version resource. Perhaps we need a similar note on the 2.3 system requirements page.

> If you are working on a Magento Commerce Cloud project, see [Service versions](https://devdocs.magento.com/cloud/project/services.html#service-versions) in the Cloud Guide.

https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html

**Comment by DominicWatts** [comment]
This pull request came from a question on whether or not 2.3 was compatible with MySQL 8

File: src/guides/v2.3/install-gde/prereq/prereq-overview.md
Comment (BarnyShergold) [reply src/guides/v2.3/install-gde/prereq/prereq-overview.md:86]: Perhaps the line should be -
Check you have the correct version of mySQL for the version of Magento you are installing, [Check Here For Supported Versions](https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html#database). The following result indicates the version you are running.

This can also be updated on the 2.4 Guide

Comment (BarnyShergold) [reply src/guides/v2.3/install-gde/prereq/prereq-overview.md]: @DominicWatts Great! Can you make the same change to the 2.4 file please? It's not symlinked

File: src/guides/v2.3/install-gde/prereq/prereq-overview.md
Comment (BarnyShergold) [inline src/guides/v2.3/install-gde/prereq/prereq-overview.md:86]:
```suggestion
Check you have the correct version of mySQL for the version of Magento you are installing, [Check Here For Supported Versions]({{ page.baseurl }}/install-gde/system-requirements-tech.html#database). The following result indicates the version you are running.
```

File: src/guides/v2.4/install-gde/prereq/prereq-overview.md
Comment (BarnyShergold) [inline src/guides/v2.4/install-gde/prereq/prereq-overview.md:82]:
```suggestion
Check you have the correct version of mySQL for the version of Magento you are installing, [Check Here For Supported Versions]({{ page.baseurl }}/install-gde/system-requirements-tech.html#database). The following result indicates the version you are running.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
I've realised the URLs need to be embedded in the manner indicated in the suggestions. Once corrected it should be good to pass.

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Pull request merged**
